components/compliance-service/reporting/relaxting/reports.go

```diff
@@ -2207,3 +2208,34 @@ func validateFiltersTimeRange(endTime string, startTime string) error {
 	}
 	return nil
 }
+
+func getStartDateFromEndDate(endTime string, startTime string) ([]string, error) {
+	if len(endTime) == 0 {
+		return nil, nil
```

Comment:
This should error out

Reply:
No, actually. When the UI sends request for 24 hours data, the end time and start time are always blank in that case.